### PR TITLE
Ensure tables initially not analyzed in TestHiveTableStatistics

### PR DESCRIPTION
**Pull request by findepi**
`TestHiveTableStatistics` needs control over whether tables are
initially analyzed, because it tests both situations: analyzed and
non-analyzed tables (and deleting stats in Hive is not possible).

### CODE REVIEW

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java [hunk 1]
```diff
@@ -45,7 +46,10 @@
         @Override
         public Requirement getRequirements(Configuration configuration)
         {
-            return mutableTable(NATION);
+            return mutableTable(
+                    HiveTableDefinition.from(NATION)
+                            .injectStats(false)
```
Comment (rschlussel2) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java]: is this what was added in the new version of tempto?

File: presto-product-tests/bin/run_on_docker.sh [hunk 2]
```diff
@@ -77,7 +77,7 @@ function docker_images_used() {
 function cleanup() {
   stop_application_runner_containers ${ENVIRONMENT}
 
-  if [[ "${LEAVE_CONTAINERS_ALIVE_ON_EXIT:-false}" = "true" ]]; then
+  if [[ "${LEAVE_CONTAINERS_ALIVE_ON_EXIT:-false}" != "true" ]]; then
```
Comment (rschlussel2) [inline presto-product-tests/bin/run_on_docker.sh]: This commit is unrelated, right?  Still seems good.  Just want to make sure I understand how these things relate to each other.

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java (hunk 1 above)
Comment (findepi) [reply presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java]: yes -- the ability to control injecting stats on per-table basis, overriding global `inject_stats_for_mutable_tables`

File: presto-product-tests/bin/run_on_docker.sh (hunk 2 above)
Comment (findepi) [reply presto-product-tests/bin/run_on_docker.sh]: Yes, unrelated. Popped up when running product tests locally. I just didn't want to run separate PR with this ;)

**Comment by kokosing** [comment]
How is that possible that these tests were passing?

**Comment by findepi** [comment]
> How is that possible that these tests were passing?

The existing tests test only partitioned table and we don't support loading stats for partitioned tables in tempto. Thus -- no stats. 
However, CBO adds tests for unpartitioned tables (the tests were passing until `inject_stats_for_mutable_tables` was changed globally) -- i will extract that into PR soon.
